jwst/tso_photometry/tso_photometry.py

```diff
@@ -289,6 +247,83 @@ def tso_aperture_photometry(
     return tbl
 
 
+def _get_int_times(datamodel):
+    """
+    Find mid times of each integration.
+
+    If the int_times table is available in the datamodel, use that.
+    If not, approximate the times using exposure metadata.
+
+    Parameters
+    ----------
+    datamodel : ~stdatamodels.jwst.datamodels.CubeModel
```

Comment:
I think these references need backticks

```suggestion
    datamodel : `~stdatamodels.jwst.datamodels.CubeModel`
```